tests/unittest/auto_deploy/singlegpu/transformations/library/test_kv_cache.py

```
@@ -489,7 +489,7 @@ def test_insert_cached_attention_rejects_unsupported_semantic_mask_backend():
         RuntimeError,
         match=(
             "Cached attention backend 'flashinfer' does not support lowering semantic mask op"
-            ".*gemma4_multimodal_mask.*Supported backends: torch, triton_paged"
+            ".*gemma4_multimodal_mask.*Supported backends: torch, triton"
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)